README.md

```diff
@@ -12,61 +12,80 @@ letting user choose from a wide selection of predefined Dockerfile snippets
 that represent those dependencies (e.g. Golang, Python, Android SDK, customized
 NPM installs).
 
-### Build
+## Build
 Make sure you setup Go acording to: https://golang.org/doc/install#install
 ```
 go get -v
 go build
 ```
 
-### Usage
+## Usage
 
+Basicaly, pazuzu cli tool has 3 steps:
+- **search** - search for available features
+- **compose** - compose **Pazuzufile** with desired features
+- **build** - create Dockerfile out of Pazuzufile
+
+## Search features
+List step is used to check what features are actually available in configured snippet provider
+```
+$ pazuzu search [regexp]
 ```
-pazuzu -h
-NAME:
-   pazuzu - Build Docker features from pazuzu-registry
-
-USAGE:
-   pazuzu [global options] command [command options] [arguments...]
-
-VERSION:
-   0.1
-
-COMMANDS:
-     build   build docker image
-     verify  verify docker image against
-     search  search for features in registry
-     list    list all features in registry
-
-GLOBAL OPTIONS:
-   --docker-endpoint value, -e value     Set the docker endpoint (default: "unix:///var/run/docker.sock")
-   --registry value, -r value            Set the registry URL (default: "http://localhost:8080/api") [$PAZUZU_REGISTRY]
-   --tokeninfo-endpoint value, -t value  Sets the OAuth2 token info URL (default: "https://token.auth.zalando.com/access_token")
-   --user value, -u value                Sets the OAuth2 user name
-   --help, -h                            show help
-   --version, -v                         print the version
+Mask is a valid regexp, i.e.:
 ```
+$ pazuzu search "node-v4.6*|java8|mvn"
+```
+Mask `node-v4.6*|java8|mvn` will use following features:
+- latest `node` version that match regexp **node-v4.6***
+- latest `java8` release available in feature snippets provider(default java8 )
 
+## Compose features
+Compose step actually creates features file out of specified 
+```
+$ pazuzu compose <space-separated-feature-names> 
 ```
-pazuzu build -h
+### Available options
+- **--with-base NAME** - specify base image
```

Comment:
As well I would keep this option in current iteration unimplemented just not to forget it in the next sprint.